src/xmipp/SConscript

```diff
@@ -141,9 +147,9 @@ else:
     python_incdirs = []
 
 # Basic libraries
-dirs = ['external','external','external','libraries','libraries','libraries','libraries','libraries']
```

Comment:
External is removed on purpose?

Reply:
I believe it's removed because `gtest` is removed a line below (i.e. as a result, `external/gtest` is removed)

Reply:
> I believe it's removed because `gtest` is removed a line below (i.e. as a result, `external/gtest` is removed)

Yes.